docs/ManualDeploymentGuideScalarDLOnAzure-1.md

```diff
@@ -0,0 +1,217 @@
+
+* [Helm](https://helm.sh/docs/intro/install/): helm command-line tool to manage releases in the AKS cluster. In this tutorial, it is used to deploy Scalar DL and Schema loading helm charts to the AKS cluster.
+   Helm version 3.2.1 or latest is required.
+
+### Requirements
+
+* You must have the authority to pull `scalar-ledger` and `scalardl-schema-loader` container images.
+* You must confirm that the replica count of the ledger and envoy pods in the `scalardl-custom-values.yaml` file is equal to the number of nodes in the `scalardlpool`.
+
+### Steps
+
+1. Download the following Scalar DL configuration files from [the scalar-kubernetes repository](https://github.com/scalar-labs/scalar-kubernetes/tree/master/conf). Note that they are going to be versioned in the future, so you might want to change the branch to use a proper version.
+    * scalardl-custom-values.yaml
+    * schema-loading-custom-values.yaml
+
+2. Update the database configuration in `scalarLedgerConfiguration` and `schemaLoading` sections as specified in [Configure Scalar DL guide](./ConfigureScalarDL.md).
+
+3. Create the docker-registry secret for pulling the Scalar DL images from the GitHub Packages.
+    
+   ```console
+    $ kubectl create secret docker-registry reg-docker-secrets --docker-server=ghcr.io --docker-username=<github-username> --docker-password=<github-personal-access-token>
+    ```
+   
+4. Run the Helm commands on the bastion to install Scalar DL on AKS.
+    
+   ```console
+    # Add Helm charts
+    $ helm repo add scalar-labs https://scalar-labs.github.io/helm-charts
+    
+    # List the Scalar charts.
+    $ helm search repo scalar-labs
+    
+    # Load Schema for Scalar DL install with a release name `load-schema`
+    $ helm upgrade --install load-schema scalar-labs/schema-loading --namespace default -f schema-loading-custom-values.yaml
+    
+    # Install Scalar DL with a release name `my-release-scalardl`
+    $ helm upgrade --install my-release-scalardl scalar-labs/scalardl --namespace default -f scalardl-custom-values.yaml
+    ```
+
+Note:
+
+* The same commands can be used to upgrade the pods.
+* Release name `my-release-scalardl` can be changed as per your convenience.
+* `helm ls -a` command can be used to list currently installed releases.
+* You should confirm the load-schema deployment has been completed with `kubectl get pods -o wide` before installing Scalar DL.
+* Follow the [Maintain Scalar DL Pods](./MaintainPods-1.md) for maintaining Scalar DL pods with Helm.
+
+## Step 5. Monitor the cluster
+
+It is critical to actively monitor the overall health and performance of a cluster running in production.
+This section shows how to configure monitoring and logging for your AKS cluster.
+
+### Steps
+
+* Enable monitoring of Azure Kubernetes Service on the basis of [the official guide](https://docs.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-enable-existing-clusters).
```

Comment:
Following the official guide will monitor what components?
I think it's a general guide so it won't monitor Scalar-specific components unless specified.
Can you clarify?

Reply:
We can only monitor general matrices such as CPU, memory etc of the pods and nodes.